Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                NewPerson();
+            }
+        }
+
+        public static void NewPerson()
```

Comment:
GetPersonData or similar should be a function name. Method names = verbs, stick to it.